elements/rh-tile/demo/rh-tile.html

```diff
@@ -0,0 +1,239 @@
+<link rel="stylesheet" href="demo.css">
+<link rel="stylesheet" href="../rh-tile-lightdom.css">
+<script type="module" src="rh-tile.js"></script>
+
+<script type="module">
+  import '@rhds/elements/lib/elements/rh-context-picker/rh-context-picker.js';
+  import '@rhds/elements/lib/elements/rh-context-provider/rh-context-provider.js';
+</script>
+
+<rh-context-picker target="demo-container"></rh-context-picker>
```

Comment:
split into separate files. context-picker etc should only be in color-context.htmj